views/css/index.css

```diff
@@ -120,9 +119,11 @@ body {
     align-items: center;
 }
 #page-container {
+    position: relative;
     flex-direction: column;
     justify-content: flex-start;
     align-items: center;
+    position: relative;
```

Comment:
`position: relative` appearing twice for this element

Reply:
Fixed